.github/workflows/check.yml

```diff
@@ -1,60 +1,50 @@
-name: "Check DVC GitHub Action"
-
+name: Check DVC GitHub Action
 on: [push]
-
 jobs:
   check:
     strategy:
       matrix:
-        runs-on: [ '{"runs-on": "ubuntu-latest"}', 
-          '{"runs-on": "macos-latest"}', 
-          '{"runs-on": "ubuntu-latest", "container": "ubuntu:18.04"}',
-          '{"runs-on": "windows-latest"}']
-
-    runs-on: ${{ fromJSON(matrix.runs-on).runs-on }}
-    container: ${{ fromJSON(matrix.runs-on).container }} 
-
+        os:
+          [
+            '{"runs-on": "ubuntu-latest"}',
+            '{"runs-on": "macos-latest"}',
+            '{"runs-on": "ubuntu-latest", "container": "ubuntu:18.04"}',
+            '{"runs-on": "windows-latest"}',
+          ]
+    runs-on: ${{ fromJSON(matrix.os).runs-on }}
+    container: ${{ fromJSON(matrix.os).container }}
     steps:
-    - uses: actions/checkout@v2
-
-    - uses: actions/setup-node@v1
-      if: "fromJSON(matrix.runs-on).container == 'ubuntu:18.04'"
-      with:
-        node-version: '12'
-
-    - uses: actions/setup-python@v2
-      if: "contains(fromJSON(matrix.runs-on).runs-on, 'windows')"
-      with:
-        python-version: '3.x'
-
-    - name: "npm ci"
-      run: npm ci
-
-    - name: "lint"
-      run: npm run lint
-
-    - name: "tests"
-      run: |
-        npm run test
-    - uses: iterative/setup-dvc@v1
-      with:
-        version: '1.0.1'
-
-    - name: "DVC test"
-      if: "!contains(fromJSON(matrix.runs-on).runs-on, 'windows')"
-      run: |
-        DVC_VER=$(dvc --version)
-        if [ $DVC_VER != '1.0.1' ]; then
-          exit 1
-        fi
-    - uses: iterative/setup-dvc@v1
-      with:
-        version: latest
-      
-    - name: "DVC test"
-      if: "!contains(fromJSON(matrix.runs-on).runs-on, 'windows')"
-      run: |
-        DVC_VER=$(dvc --version)
-        if [ $DVC_VER == '1.0.1' ]; then
-          exit 1
-        fi
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        if: "fromJSON(matrix.os).container == 'ubuntu:18.04'"
+        with:
+          node-version: '12'
+      - uses: actions/setup-python@v2
+        if: "contains(fromJSON(matrix.os).runs-on, 'windows')"
+        with:
+          python-version: '3.x'
+      - run: npm ci
+      - run: npm run lint
+      - run: npm run test
+      - name: run local action with '1.0.1'
+        uses: ./github/..
```

Comment:
note: in the past I've found `./` sometimes doesn't work so this is a work-around to force using the local action.

Reply:
[Seems to work for the official templates](https://github.com/actions/javascript-action/blob/2498a9f3f2a667476bb92072bc0e3fd9634572bc/.github/workflows/test.yml#L23), but... 🤷‍♂️ 
Would it be useful to add a comment along with that line?

Reply:
removed - seems to be working